scripts/release.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+indent() {
+    RE="s/^/       /"
+    [ $(uname) == "Darwin" ] && sed -l "$RE" || sed -u "$RE"
+}
+
+MANAGE_FILE=$(find . -maxdepth 3 -type f -name 'manage.py' | head -1)
+MANAGE_FILE=${MANAGE_FILE:2}
```

Comment:
Nit: can you include a comment that this is just trimming the `./` from the path?